support-frontend/assets/pages/contributions-landing/components/ContributionForm.jsx

```diff
@@ -256,6 +257,7 @@ function withProps(props: PropTypes) {
           isTestUser={props.isTestUser}
           country={props.country}
         />
+        <div id="robot_checkbox" className={props.paymentMethod === 'Stripe' ? 'robot_checkbox' : 'hidden'} />
```

Comment:
To add an error message, we can wrap this in a `div` and display an error if `checkoutFormHasBeenSubmitted && !v2IsLowRisk`